index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+    <title>My Portfolio</title>
+</head>
+<body>
+    
+    <!-- navbar section -->
+<nav class="navbar">
+    <h3 id="logo">Dela McRaph</h3>
+    <ul>
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+    </ul>
+
+</nav>

```

Comment:
- [ ] Kindly always adhere to the project requirement, as I can see your `UI` is not matching with the `Figma` template `mobile` design as the required milestone feature. It is very important to always fulfill the core requirement as a *software developer*. Please take a look at the below images.

### Your Design
![Screenshot from 2023-04-27 19-30-30](https://user-images.githubusercontent.com/66476329/234885630-3976f35e-8b0c-450c-9f78-fb7fec53cb60.png)

### Design should look like

![Screenshot from 2023-04-27 19-32-53](https://user-images.githubusercontent.com/66476329/234886251-e9ba315d-dcaf-4033-8023-79acc377eea9.png)


- [ ] Kindly follow the correct `HTML` semantics as per requirement. You must follow the best practices for `HTML/CSS` because it will improve your website accessibility which helps those people who are visually impaired. Adding correct semantics would help screen reader to further the visual impaired people to navigate through your website easily. Please look at the below image.

![Screenshot from 2023-04-27 19-33-55](https://user-images.githubusercontent.com/66476329/234887358-98caa9f4-8bb0-45d4-9eac-523b4770d876.png)